examples/asset/utilization/assetUtilization.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NationalInstruments.SystemLink.Clients.AssetManagement" Version="1.0.0" />
```

Comment:
This needs to reference an actual NuGet version, even if it's not published to nuget.org yet.